src/main/java/com/volasoftware/tinder/services/UserService.java

```diff
@@ -32,30 +24,28 @@ public class UserService {
     private final VerificationRepository verificationRepository;
     private final ResourceLoader resourceLoader;
     private JavaMailSender mailSender;
```

Comment:
Do we use that?

Reply:
I've already removed the 26th and 25th line, but I'm using the verification repository to save the token on line 62